lib/daru/io/io.rb

```diff
@@ -183,6 +183,18 @@ def load filename
         end
       end
 
+      def from_html path, opts
+        page = Mechanize.new.get(path)
+        page.search('table').map { |table| html_parse_table table }
+            .keep_if { |table| html_search table, opts[:match] }
+            .compact
+            .map { |table| html_decide_values table, opts }
+            .map { |table| html_table_to_dataframe table }
+      rescue LoadError
+        STDERR.puts '\nInstall the mechanize gem version 2.7.5 for using'\
+        ' from_html function.'
```

Comment:
Acknowledged, does a run-time error like `raise RuntimeError, 'Install the mechanize gem 2.7.5 for using from_html function'` work?